src/Features/CSharp/Portable/QuickInfo/CSharpSemanticQuickInfoProvider.cs

```diff
@@ -86,8 +86,19 @@ protected override ImmutableArray<TaggedText> TryGetNullabilityAnalysis(Workspac
                 return default;
             }
 
+            switch (symbolInfo.Symbol)
+            {
+                case IFieldSymbol { HasConstantValue: true }: return default;
+                case ILocalSymbol { HasConstantValue: true }: return default;
+            }
+
             var typeInfo = semanticModel.GetTypeInfo(bindableParent, cancellationToken);
 
+            if (typeInfo.Type?.IsValueType == true)
```

Comment:
Consider sticking a comment to explain why (since we think it's more noise in this case...)